src/app/components/forms/user-details-form/user-details-form.component.ts

```diff
@@ -33,6 +35,11 @@ export class UserDetailsFormComponent implements OnInit {
   ngOnInit() {
     this.initSettingsForm();
     this.initPasswordForm();
+    this.loadData();
+  }
+
+  loadData() {
```

Comment:
Extend TranslationsListener. See conventions please.
Its not enough to just add loadData function.